scaler/io/ymq/bytes.h

```diff
@@ -21,31 +25,53 @@ class Bytes {
         if (_tag != Owned)
             return;
 
-        if (is_empty())
-            return;
-
         delete[] _data;
-        this->_data = NULL;
     }
 
     Bytes(uint8_t* m_data, size_t m_len, Ownership tag): _data(m_data), _len(m_len), _tag(tag) {}
 
 public:
-    // move-only
-    // TODO: make copyable
-    Bytes(const Bytes&)            = delete;
-    Bytes& operator=(const Bytes&) = delete;
+    // TODO: Figure out what should this tag do
+    Bytes(char* data, size_t len, Ownership tag = Ownership::Owned)
+        : _data(datadup((uint8_t*)data, len)), _len(len), _tag(tag) {}
+
+    Bytes(): _data {}, _len {}, _tag {} {}
```

Comment:
Better to be explicit?

```suggestion
    Bytes(): _data {nullptr}, _len {0}, _tag {Ownership::Owned} {}
```

Reply:
I'm not -- let's get rid of it